### DISCUSSION TITLE
Status for Combobox with Multi-select

### DISCUSSION THREAD

**Discussion by jstwilio** [twilio-labs/paste#2279] [General] [answered]
Hello there - My team (Data Platform Report Center) is building out an Insights feature (Target completion date - 4/30) and we need the "search combo box with multi-select option".  We are blocked at the moment. The [roadmap published here](https://paste.twilio.design/roadmap/) indicates it's at risk - first line item in list.  Can someone help us understand the due date and next steps? Who is the right point of contact for this feature? Thanks cc: [@sfalkoff](https://twilio.slack.com/team/WJ2PGM56W), [@jlittleton](https://twilio.slack.com/team/WJE7Q8W1W), [@skolla](https://twilio.slack.com/team/U02NLP894AZ)

**Comment by shleewhite** [comment]
Hi @jstwilio! 

Currently, the design is at risk for this release (which finishes on March 22) but it will completed for next release. The engineering work will start once the design is finalized.

If you need to use a multiselect combobox before then, you can use the combobox primitive with the form pill group component. We have an example of how to do that on [the Combobox Primitive page](https://paste.twilio.design/primitives/combobox-primitive#multiselect-combobox-example).

Please let me know if you have any other questions!

**Comment by jstwilio** [comment]
Thanks.  For our planning, can we get a sense of when the combo-box with
multi-select will be available for teams to use?  And can we kept in loop
of that development?

Thanks
Jyotsna

**Reply by shleewhite** [reply]
The engineering work will probably happen within the next 2 or 3 releases, but it is not finalized. We will follow up in this thread when the component is available.

**Reply by shleewhite** [reply]
For your tracking, here is the Jira ticket: https://issues.corp.twilio.com/browse/DSYS-2412

**Comment by jstwilio** [comment]
Hello - Following up on this.

(1) What's the best way to keep up with this work since it is a
release blocker for our Enterprise/Org Insights feature release?  I looked
up https://issues.corp.twilio.com/browse/SIT-34724 but don't see any updates
(2) Can you please provide an estimated time of availability?
(3) We have created a dependency for your team on Airtable
<https://airtable.com/appChlFldwVpI3TIL/tbl5fI1QQsMEj63Ll/viwnmk3ewkJPlFxbe/recRBVSLhy3JdGO6g?blocks=hide>
so we can track cross-org, cross-team dependencies.  Please let me know if
you all any questions or concerns

**Reply by TheSisb** [reply]
Hi @jstwilio,

We don't currently have a new estimate for that component due to the HAP.  It may be best to get in touch via email to discuss further.

**Comment by jstwilio** [comment]
Surely.  Who should I email?  I can followup

Thanks
Jyotsna

**Comment by SiTaggart** [comment, marked as answer]
Hi @jstwilio 

Multi select combo was recently released as part of the Combobox package. https://paste.twilio.design/components/multiselect-combobox

Let us know how you get on.
